eng/code-quality-reports/src/main/resources/revapi/revapi.json

```
@@ -418,9 +418,9 @@
         },
         {
           "regex": true,
-          "code": "java\\.annotation\\.(attributeRemoved|attributeAdded)",
-          "old": ".*? com\\.azure\\.resourcemanager\\.(cosmos|network|mobilenetwork)\\.models.*",
-          "justification": "Jackson annotation changed."
+          "code": "java\\.annotation\\.(attributeRemoved|attributeAdded|removed)",
+          "old": ".*? com\\.azure\\.resourcemanager\\.(*)\\.models.*",
```

Please, this is just a regex. Test it before commit.

You likely missed the `.` on `.*`
```suggestion
          "old": ".*? com\\.azure\\.resourcemanager\\..*\\.models.*",
```